templates/patterns/card/card.ui_patterns.yml

```diff
@@ -1,119 +1,66 @@
+      preview: 'Title card'
     subtitle:
-      type: "array"
-      label: "Subtitle"
-      description: "Card subtitle [content, tag, classes]"
-      preview:
-        content: "Subtitle card"
-        classes: "mb-2"
+      type: text
+      label: 'Subtitle'
+      description: 'Card subtitle.'
+      preview: 'Subtitle card'
     text:
-      type: "array"
-      label: "Text"
-      description: "Card text [content, tag, classes]"
-      preview:
-        content: "Text card with longer text which is supported"
-        classes: "mb-2"
+      type: text
+      label: 'Text'
+      description: 'Card text.'
+      preview: 'Text card with longer text which is supported'
     image:
-      type: "array"
-      label: "Image"
-      description: "Card image [path, alt, position]"
+      type: array
+      label: 'Image'
+      description: 'Card image with path, alt and position.'
       preview:
-        path: "https://placeimg.com/1000/500/people"
-        alt: "Alternative text for card image"
-        position: "top"
-        rounded: 3
+        path: 'https://placeimg.com/1000/500/people'
+        alt: 'Alternative text for card image'
     content:
-      type: "render"
-      label: "Content"
-      description: "Card content"
-      preview: "<button class='btn btn-primary'>Custom action</button>"
-    card_header:
-      type: "render"
-      label: "Card header"
-      description: "Card header content"
-      preview: "Header of card"
-    card_footer:
-      type: "render"
-      label: "Card footer"
-      description: "Card footer content"
-      preview: "Footer of card"
-    horizontal_grid:
-      type: "array"
-      label: "Horizontal grid"
-      description: "Horizontal grid layout [left_col, right_col, gutter]"
-    extra_classes_body:
-      type: "text"
-      label: "Extra classes for card body"
-      preview: "body-class"
-    extra_classes_header:
-      type: "text"
-      label: "Extra classes for card header"
-      preview: "header-class"
-    extra_classes_footer:
-      type: "text"
-      label: "Extra classes for card footer"
-      preview: "footer-class"
+      type: render
+      label: 'Content'
+      description: 'Card content.'
+      preview: '<button class="btn btn-primary">Custom action</button>'
+    header:
+      type: text
+      label: 'Card header'
+      description: 'Card header content.'
+      preview: 'Header of card'
+    footer:
+      type: text
+      label: 'Card footer'
+      description: 'Card footer content.'
+      preview: 'Footer of card'
     badges:
-      type: "array"
-      label: "Badges for the card"
+      type: array
+      label: 'Badges'
+      description: 'List of badges for the card.'
       preview:
-        - label: "Primary badge"
-        - label: "Secondary badge"
-          assistive_text: "Assistive text for secondary badge"
+        - label: 'Primary badge'
+        - label: 'Secondary badge'
           rounded_pill: true
     date:
```

Comment:
The date looks horrible, let's remove it for now and we will add it when it's styled properly.